int32 lift_set_relative
int32 tilt_set_relative

uint16 lift_set_absolute
uint16 tilt_set_absolute

bool measure
bool tare

int16 cal_factor